Dockerfile.dapper

```diff
@@ -109,7 +110,7 @@ ENV DAPPER_ENV REPO TAG DRONE_TAG DRONE_COMMIT DRONE_BRANCH DRONE_BUILD_EVENT SY
 ENV DAPPER_SOURCE /go/src/github.com/rancher/rancher/
 ENV DAPPER_OUTPUT ./bin ./dist ./go.mod ./go.sum ./pkg/apis/go.mod ./pkg/apis/go.sum ./pkg/client/go.mod ./pkg/client/go.sum ./scripts/package ./pkg/settings/setting.go ./package/Dockerfile ./Dockerfile.dapper
 ENV DAPPER_DOCKER_SOCKET true
-ENV DAPPER_RUN_ARGS "-v rancher2-go16-pkg-1:/go/pkg -v rancher2-go16-cache-1:/root/.cache/go-build --privileged"
+ENV DAPPER_RUN_ARGS "-v rancher2-go16-pkg-1:/go/pkg -v rancher2-go16-cache-1:/root/.cache/go-build --privileged --add-host=host.docker.internal:172.17.0.1 --network=host"
```

Comment:
There is no description in the issue that explains why this is needed

Reply:
My apologies, so the k3d cluster runs in a docker container, so when running k3d in a DinD environment there were issues with binding to the default 0.0.0.0 address. Highlighted by this ticket: https://github.com/k3d-io/k3d/issues/1008 that is the reason for this flag: `--add-host=host.docker.internal:172.17.0.1`

`--network=host` is when we import the k3d cluster into rancher, the k3d cluster need to know its from the IP address of the host machine, again because running in a DinD environment.

Reply:
After some testing ` --add-host=host.docker.internal:172.17.0.1` is not necessary. Having `--network=host` is enough to be able to bind the k3d cluster to the local address and then be able to import the k3d cluster into rancher.